src/com/shop/Shop.java

```diff
@@ -1,324 +1,335 @@
+			updateQuantity = conn.prepareStatement("UPDATE Sales SET quantity=quantity-1 WHERE pid= ? AND cid= ?");
+			selectProductsName = conn.prepareStatement("SELECT * FROM Products WHERE name LIKE ?");
+			selectTotalCost = conn.prepareStatement(
+					"SELECT P.pid AS pid, P.name AS name, P.price*S.quantity AS total FROM Products P, Sales S WHERE S.pid=P.pid AND S.cid= ?");
+			selectAllCustomers = conn.prepareStatement("SELECT * FROM Customers");
+			insertCustomerRow = conn.prepareStatement("INSERT INTO Customers (cid, name, budget) VALUES (?, ?, ?)");
+		} catch (SQLException e) {
+			e.printStackTrace();
+		}
+	}
+
+	private Connection connect() {
+		Connection conn = null;
+		
+		// Connect To Database
+		try {
+			// Load Driver - jdbcDriver
+			Class.forName("oracle.jdbc.OracleDriver");
+			conn = DriverManager.getConnection("jdbc:oracle:***********", "*******", "*******");
+			System.out.println("Connection Successful");
+		} catch (SQLException e) {
+			System.out.println("Connection ERROR");
+			e.printStackTrace();
+		} catch (ClassNotFoundException e1) {
+			e1.printStackTrace();
+		}
+		
+		return conn;
+	}
+
+	private void displayMenu() {
+		System.out.println("\n===== MAIN MENU =====");
+		System.out.println("P - List all products.");
+		System.out.println("O – Order a product.");
+		System.out.println("R – Return a product.");
+		System.out.println("S - Search a product by name.");
+		System.out.println("E - List all orders.");
+		System.out.println("C - Current budget.");
+		System.out.println("X - Exit.");
+		System.out.print("\nEnter Menu Option: ");
+	}
+
+	private boolean isCustomerVerified() {
+		try {
+			selectCustomerRow.setInt(1, id);
+			queryResult = selectCustomerRow.executeQuery();
+			
+			if (queryResult.next()) {
+				return true;
+			}
+		} catch (SQLException e) {
+			e.printStackTrace();
+		}
+		
+		return false;
+	}
+
+	private void menuP() {
+		try {
+			selectProductsName.setString(1, "%");
+			queryResult = selectProductsName.executeQuery();
+			System.out.printf("%-5s | %-20s | %-10s%n", "ID", "NAME", "PRICE");
+			
+			while (queryResult.next()) {
+				System.out.printf("%-5d | %-20s | %.2f%n", queryResult.getInt("pid"), queryResult.getString("name"),
+						queryResult.getDouble("price"));
+			}
+		} catch (SQLException e) {
+			e.printStackTrace();
+		}
+	}
+
+	private void menuO() {
+		System.out.print("Enter Product ID: ");
+		int productId = input.nextInt();
+		System.out.print("\nEnter Quantity: ");
+		int productQuantity = input.nextInt();
+		System.out.println();
+		
+		try {
+			selectProductsRow.setInt(1, productId);
+			queryResult = selectProductsRow.executeQuery();
+			
+			if (queryResult.next()) {
+				int total = queryResult.getInt("price") * productQuantity;
+				selectSalesRow.setInt(1, id);
+				selectSalesRow.setInt(2, productId);
+				queryResult = selectSalesRow.executeQuery();
+				
+				if (queryResult.next()) {
```

Comment:
Added new line whitespace for better readability between if statements, loops, and regular statements